surveys/2024-annual-survey/questions.md

```diff
@@ -414,11 +382,33 @@ Type: select all that apply (optional)
 - Achieving structured concurrency with async code
 - Writing executor-agnostic async code
 - Borrow checker not allowing valid code
+
+- (From here the additional one)
+- Runtime performance
+- Compile times
+- Binary size (of the final compiled artifact)
```

Comment:
"Achieving small binary size of compiled artifacts" or "Rust binaries are too large".